src/Client/ConcordiumClient.cs

```diff
@@ -0,0 +1,172 @@
+using Concordium.Sdk.Transactions;
+
+using Grpc.Core;
+using Grpc.Net.Client;
+
+namespace Concordium.Sdk.Client;
+
+/// <summary>
+/// A client for interacting with the Concordium gRPC API V2 exposed by nodes.
+/// </summary>
+public class ConcordiumClient : IDisposable
+{
+    /// <summary>
+    /// The raw client.
+    /// </summary>
+    public RawClient Raw { get; init; }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Client"/> class.
+    ///
+    /// Optionally use <paramref name="channelOptions"/> to specify connection settings
```

Comment:
@abizjak I changed this to take a `GrpcChannelOptions`instance which can be used to adjust and configure various connection related parameters.